src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss

```diff
@@ -1,22 +1,103 @@
 ﻿@import "../../Styles/Fluent/colors.fluent.scss", "../../Styles/functions.scss";
 
-.bit-brc-fluent li {
-    display: inline-flex;
-}
+.bit-brc-items-wrapper {
+
+    ul {
+        padding-inline-start: 0;
+
+        li {
+            display: inline-flex;
+            align-items: center;
+        }
+    }
+
+    button {
+        font-size: rem(20px);
+        font-weight: 400;
+        color: $Grey160;
+        background-color: transparent;
+        cursor: pointer;
+
+        &:hover {
+            background-color: $Grey20;
+        }
+    }
 
-.bit-brc-fluent ul {
-    padding-inline-start: 0;
+    .bit-brc-itm {
+        font-size: rem(20px);
+        padding: 0 rem(8px);
+        color: $Grey90;
+    }
+
+    .bit-brc-lst-itm {
+        font-size: rem(20px);
+        font-weight: 600;
+        padding: 0 rem(8px);
+        color: $Grey160;
+    }
 }
 
-.bit-brc-itm-fluent {
-    font-size: rem(20px);
-    padding: 0 rem(8px);
-    color: $Grey90;
+.bit-brc-overlay {
+    display: none;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 999;
+    background-color: transparent;
 }
 
-.bit-brc-lst-itm-fluent {
-    font-size: rem(20px);
-    font-weight: 600;
-    padding: 0 rem(8px);
-    color: $Grey160;
+.bit-brc-overflow-menu-wrapper {
+    display: none;
+    position: fixed;
+    max-height: 744px;
```

Comment:
Plz, use rem(744px) instead of 744px

Reply:
Has this issue been solved? @msangtarash